libs/imap/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2017 Lucian Cristian <lucian.cristian@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+# updated to work with latest source from abrasive
```

Comment:
I don't understand this comment. Can we drop it, or is it important?

Reply:
I started from some template, I'll clean up